models/salesforce_marketing_cloud__sends_overview.sql

```diff
@@ -35,7 +35,7 @@ with sends as (
 
 ), joined as (
   select
-    sends_aggs.*,
+    sends_aggs.*, 
```

Comment:
Same comment about some of the first fields the customer sees in this model are `_fivetran_synced` and `_fivetran_deleted` which don't add much value in the model. We should consider either removing the fields from this model or moving them to the end of the staging model.